README.md

```diff
@@ -1,3 +1,122 @@
-# vscode
+# VS Code
 
-The action runs VSCode in web browser.
+This action runs an instance of [VS Code](https://code.visualstudio.com/) and runs in in the default web browser.
+
+It requires the name of the image on which to run VS Code and references to the following 5 volumes: data, code, config, notebooks, and results. 
+These volumes will be mounted to /project/data, /project/modules, /project/config, /project/notebooks and /project/results respectively.
+
+### Quick example
+
+```
+jobs:
+  vscode:
+    action: gh:neuro-actions/vscode@master
+    args:
+      image: neuromation/base
```

Comment:
In `v1.0.1` this argument is optional and will be equal to `neuromation/base` by default. We can remove it from the example.

Reply:
Fixed in the latest commit